example_configs/presto.yml

```diff
@@ -0,0 +1,48 @@
+lowercaseOutputName: true
+lowercaseOutputLabelNames: true
+whitelistObjectNames: ["com.facebook.presto.hive:type=NamenodeStats,*", "com.facebook.presto.hive.metastore:type=ThriftHiveMetastore,*", "com.facebook.presto.failureDetector:name=HeartbeatFailureDetector,*", "com.facebook.presto.execution:name=QueryManag\
```

Comment:
Does this make a large enough difference to be worth maintaining it as new mbeans are added?

Reply:
Yeah that's a good question. We are explicitly pattern matching the beans we are interested in so only those seem to be making it through. I'll drop this in favor of the rules.